### PR TITLE
Restaurant Page Pull request

### PR DESCRIPTION
**Pull request by NataJenkins**
Page build with html, css and Javascript  of a restaurant page in which you can navigate through tabs

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Invalid Submission

### Hi @NataJenkins :wave: 

I have marked this submission as invalid, this is because your last commit failed linter validation. Passing linter validation is a hard requirement. Kindly refer to this [link](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-#) for more details.

Ensure your commits pass linter validations, then submit another form for a code review. 

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @NataJenkins :wave: 

There are few updates to make:

* [ ] Webpack-dev-server should be installed as devDependencies in package.json(
[screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

In your package.json the **scripts** property should look like so:


```
    "scripts": {
        "dev": "webpack --mode development",
        "build": "webpack --mode production",
        "start": "webpack-dev-server --mode development --open" // optional
      },
```
* [ ] When Navigating the application, if I click on the **home menu**, the navigation bar disappears but re-emerges when I reload the page. Please do fix that.

Make these changes, then submit another form for a code review.


### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// import "./styless.css";
```
Comment (rukundoeric) [inline src/index.js]: remove this comment

File: src/tabs.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function onTabClick(event) {
+  const activeTabs = document.querySelectorAll('.active');
+
+  // deactivate tab
+  activeTabs.forEach((tab) => {
+    tab.className = tab.className.replace('active', ' ');
+  });
+
+  // activate new tab
+  //   event.target.parentElement.className += " active";
```
Comment (rukundoeric) [inline src/tabs.js]: remove this comment

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - [x]  Kindly remove all commented lines of code before you merge.

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**